chapters/interfaces.txt

```diff
@@ -2752,9 +2752,16 @@ code:PatchVertices::
 Decorating a variable with the code:PatchVertices built-in decoration will
 make that variable contain the number of vertices in the input patch being
 processed by the shader.
-A single tessellation control or tessellation evaluation shader can: read
+In a Tessellation Control Shader, this is the same as the name:patchControlPoints
+member of slink:VkPipelineTessellationStateCreateInfo.
+In a Tessellation Evaluation Shader, code:PatchVertices is equivalent to the
```

Comment:
```suggestion
In a Tessellation Evaluation Shader, code:PatchVertices is equal to the
```

Just a minor quibble, but I'd prefer to say "equal" here to emphasize that the value is the same but they're not "equivalent" because they have different uses. You can't substitute one for the other.

Reply:
Shouldn't we then just say `code::PatchVertices is the same as the tessellation control output patch size`?

Reply:
On a second thought, I think "is equal to" sounds better.

Reply:
They're not the same thing, but they have the same value, so I'm not sure saying that they are "the same" is clearer. I think saying that they're equal probably conveys that best.

Reply:
Agreed.